general/development/process.md

```diff
@@ -292,7 +292,7 @@ Decisions will be posted on the issue and that issue will be closed, allowing an
 ## See also
 
 - [Release process](https://docs.moodle.org/dev/Release_process)
-- [Deprecation](https://docs.moodle.org/dev/Deprecation)
+- [Deprecation](general/development/policies/deprecation)
```

Comment:
I think this should be `/general/development/policies/deprecation` - missing leading `/`?

Was this autogenerated?

Reply:
You're right! 
I don't think it was autogenerated (so probably I missed it!).